data/856/326/65/85632665.geojson

```diff
@@ -110,10 +251,10 @@
     "wof:tags":[]
 },
   "bbox": [
-    43.13869476300005,
-    12.71036148100006,
-    43.15783309900013,
-    12.72124958000006
+    43.138695,
+    12.710361,
+    43.157833,
+    12.72125
 ],
-  "geometry": {"coordinates":[[[[43.15202713000008,12.71913909900007],[43.15286254900008,12.71833324400006],[43.15286254900008,12.71500015300006],[43.15119552600004,12.71333313000002],[43.15080642700008,12.71119403800003],[43.14658355700004,12.71122264900004],[43.14494323700006,12.71036148100006],[43.14324951200007,12.71122264900004],[43.14288711500006,12.71247196200005],[43.13872146600005,12.71663856500004],[43.13869476300005,12.71916675600005],[43.13991546600005,12.72041702300004],[43.14241790800003,12.72038841200003],[43.14408493000008,12.71875],[43.14494323700006,12.71875],[43.14827728300003,12.72122192400008],[43.14988708500005,12.72124958000006],[43.15202713000008,12.72000026700005],[43.15202713000008,12.71913909900007]]],[[[43.15783309900013,12.71577739700012],[43.15661239600013,12.71458339700013],[43.15565086428472,12.71755256226152],[43.15372085600018,12.71913909900007],[43.15494537400008,12.71955585500012],[43.15705490100009,12.71749973300018],[43.15700149500009,12.71663856500015],[43.15783309900013,12.71577739700012]]]],"type":"MultiPolygon"}
+  "geometry": {"coordinates":[[[[43.152027,12.719139],[43.152863,12.718333],[43.152863,12.715],[43.151196,12.713333],[43.150806,12.711194],[43.146584,12.711223],[43.144943,12.710361],[43.14325,12.711223],[43.142887,12.712472],[43.138721,12.716639],[43.138695,12.719167],[43.139915,12.720417],[43.142418,12.720388],[43.144085,12.71875],[43.144943,12.71875],[43.148277,12.721222],[43.149887,12.72125],[43.152027,12.72],[43.152027,12.719139]]],[[[43.157833,12.715777],[43.156612,12.714583],[43.155651,12.717553],[43.153721,12.719139],[43.154945,12.719556],[43.157055,12.7175],[43.157001,12.716639],[43.157833,12.715777]]]],"type":"MultiPolygon"}
```

Comment:
Geom needs some love, the low rez we have now cuts across one of the component islands :\